testdata/directive_words.txt

```diff
@@ -1,12 +1,14 @@
 # Ignore words in directive comments.
 ! gospel -show=false -ignore-idents=false
-stdout 'main.go:5:1: "gennerated" is misspelled in comment'
-stdout 'main.go:5:1: "sumtype" is misspelled in comment'
-stdout 'main.go:5:1: "ignoretypes" is misspelled in comment'
+stdout 'main.go:3:55: "gennerated" is misspelled in comment'
+stdout 'main.go:5:12: "gennerated" is misspelled in comment'
+stdout 'main.go:6:6: "sumtype" is misspelled in comment'
+stdout 'main.go:6:14: "ignoretypes" is misspelled in comment'
```

Comment:
FYI, at this point you might as well use `cmp stdout some-file`; it will be less repetitive, you can automatically update the txtar file with testscript's UpdateScripts feature, and it will catch if the output has extra files you didn't expect (as you currently only test a subset)

for example: https://github.com/mvdan/sh/blob/ffec228c8359a56fd65a72f87efb7517307204f5/cmd/shfmt/testdata/scripts/walk.txt#L60-L96

Reply:
Thanks, yes, I saw that and have started using it in the indent test. I'll rework these to use that.